### ISSUE TITLE
split /updates API and package information

### ISSUE THREAD

**Issue by jdobes** [RedHatInsights/vmaas#266]
Currently, there are summary and description fields filled for each package having an update in /updates response. This information should be provide by separate API similar to /errata.

**Comment by ggainey** [comment]
Not sure why, but duplicate repositories seem to be returned.

Example NEVRA: `kernel-2.6.32-131.0.15.el6.x86_64`

Returns:
```json
{
  "package_list": {
    "kernel-2.6.32-131.0.15.el6.x86_64": {
      "summary": "The Linux kernel",
      "description": "The kernel package contains the Linux kernel (vmlinuz), the core of any\nLinux operating system.  The kernel handles the basic functions\nof the operating system: memory allocation, process allocation, device\ninput and output, etc.",
      "repositories": [
        "rhel-6-workstation-rpms",
        "rhel-6-workstation-rpms",
        "rhel-6-server-rpms",
        "rhel-6-server-rpms"
      ]
    }
  }
}
```

DB was loaded with these repos:

```json
[{
  "entitlement_cert": {
    "name": "RHSM-CDN",
    "ca_cert": "..."
   "key": ""
  },
  "products": {
    "Red Hat Enterprise Linux Server": {
      "redhat_eng_product_id": 69,
      "content_sets": {
        "rhel-7-server-rpms": {
          "name": "Red Hat Enterprise Linux 7 Server (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/server/7/$releasever/$basearch/os/",
          "basearch": ["x86_64"],
          "releasever": ["7Server", "7.4"]
        },
        "rhel-7-server-optional-rpms": {
          "name": "Red Hat Enterprise Linux 7 Server - Optional (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/server/7/$releasever/$basearch/optional/os/",
          "basearch": ["x86_64"],
          "releasever": ["7Server", "7.4"]
        },
        "rhel-6-server-rpms": {
          "name": "Red Hat Enterprise Linux 6 Server (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/server/6/$releasever/$basearch/os/",
          "basearch": ["x86_64"],
          "releasever": ["6Server", "6.9"]
        },
        "rhel-6-server-optional-rpms": {
          "name": "Red Hat Enterprise Linux 6 Server - Optional (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/server/6/$releasever/$basearch/optional/os/",
          "basearch": ["x86_64"],
          "releasever": ["6Server", "6.9"]
        }
      }
    },
    "Red Hat Enterprise Linux Workstation": {
      "redhat_eng_product_id": 71,
      "content_sets": {
        "rhel-7-workstation-rpms": {
          "name": "Red Hat Enterprise Linux 7 Workstation (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/workstation/7/$releasever/$basearch/os/",
          "basearch": ["x86_64"],
          "releasever": ["7Workstation", "7.4"]
        },
        "rhel-7-workstation-optional-rpms": {
          "name": "Red Hat Enterprise Linux 7 Workstation - Optional (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/workstation/7/$releasever/$basearch/optional/os/",
          "basearch": ["x86_64"],
          "releasever": ["7Workstation", "7.4"]
        },
        "rhel-6-workstation-rpms": {
          "name": "Red Hat Enterprise Linux 6 Workstation (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/workstation/6/$releasever/$basearch/os/",
          "basearch": ["x86_64"],
          "releasever": ["6Workstation", "6.9"]
        },
        "rhel-6-workstation-optional-rpms": {
          "name": "Red Hat Enterprise Linux 6 Workstation - Optional (RPMs)",
          "baseurl": "https://cdn.redhat.com/content/dist/rhel/workstation/6/$releasever/$basearch/optional/os/",
          "basearch": ["x86_64"],
          "releasever": ["6Workstation", "6.9"]
        }
      }
    }
  }
},
{
  "products": {
    "Fedora": {
      "content_sets": {
        "updates": {
          "name": "Fedora Updates",
          "baseurl": "https://mirrors.nic.cz/fedora/linux/updates/$releasever/$basearch/",
          "basearch": ["x86_64"],
          "releasever": ["27"]
        }
      }
    }
  }
}]
```

**Comment by eherget** [comment]
This is fixed by PR #294
